### PR TITLE
Set proper cmake variables.

### PR DESCRIPTION
**Pull request by ahoereth**
```
-- open62541 found with version 1.3.5
-- open62541cpp found with version 1.2.666: /home/ahoereth/repos/autocrane-core/deps/include /home/ahoereth/repos/autocrane-core/deps/lib/libopen62541cpp.so
```

### CODE REVIEW

**Comment by ahoereth** [comment]
Required to properly handle multiple versions on a single machine apparently

**Comment by ahoereth** [comment]
https://github.com/psiori/autocrane-core/pull/3184

File: src/open62541cppConfig.cmake.in [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
-
 @PACKAGE_INIT@
 
-set_and_check(OPEN62541CPP_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
-set_and_check(OPEN62541CPP_LIB_DIR "@PACKAGE_LIB_INSTALL_DIR@")
+set_and_check(open62541cpp_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
+set_and_check(open62541cpp_INSTALL_DIR "@PACKAGE_LIB_INSTALL_DIR@")
+
+set_and_check(open62541cpp_LIBRARY "@PACKAGE_LIB_INSTALL_DIR@/libopen62541cpp${CMAKE_SHARED_LIBRARY_SUFFIX}")
+# set_and_check(open62541cpp_LIBRARY "@PACKAGE_LIB_INSTALL_DIR@/libopen62541cpp${CMAKE_STATIC_LIBRARY_SUFFIX}")
```
Comment (sergioherrera7) [inline src/open62541cppConfig.cmake.in:7]:
```suggestion
```

Comment (ahoereth) [reply src/open62541cppConfig.cmake.in:7]: Want to keep it if we ever want static linking

**Pull request merged**